src/components/DiscoverSentance/DiscoverSentance.jsx

```
@@ -37,10 +38,13 @@ const SpeakSentenceComponent = () => {
   const [totalSyllableCount, setTotalSyllableCount] = useState("");
   const [isNextButtonCalled, setIsNextButtonCalled] = useState(false);
 
+  const levelCompleteAudioSrc = usePreloadAudio(LevelCompleteAudio);
+
   const callConfettiAndPlay = () => {
-    let audio = new Audio(LevelCompleteAudio);
+    let audio = new Audio(levelCompleteAudioSrc);
     audio.play();
     callConfetti();
+    window.telemetry?.syncEvents && window.telemetry.syncEvents();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)